tests/schema/tests.py

```diff
@@ -1749,6 +1750,20 @@ def test_indexes(self):
             self.get_indexes(Book._meta.db_table),
         )
 
+    def test_func_index(self):
+        func_index = Index(fields=[Lower('title').desc()], name='title_lower_idx')
+        with connection.schema_editor() as editor:
+            sql = func_index.create_sql(Book, editor)
+
+        sql = sql.upper()
+        self.assertIn('LOWER', sql)
```

Comment:
Given that the index is named `title_lower_id` this `LOWER` check could check against that as well as `lower(xxx)` -- please rename the index so there is only one match for `LOWER` (same for `TITLE`)